pkg/loadbalancer/server.go

```diff
@@ -161,11 +161,10 @@ func (s *Server) EnsureLoadBalancerDeleted(ctx context.Context, clusterName stri
 
 // loadbalancer name is a unique name for the loadbalancer container
 func loadBalancerName(clusterName string, service *v1.Service) string {
-	hash := sha256.Sum256([]byte(loadBalancerSimpleName(clusterName, service)))
-	encoded := base32.StdEncoding.EncodeToString(hash[:])
-	name := constants.ContainerPrefix + "-" + encoded[:40]
-
-	return name
+	h := sha256.New()
+	io.WriteString(h, loadBalancerSimpleName(clusterName, service))
```

Comment:
```suggestion
	_ = io.WriteString(h, loadBalancerSimpleName(clusterName, service))
```
to avoid the linter error